src/plugins/timelion/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
__note:__ we need to get access to the `timelion.ui.enabled` setting so introducing NP timelion plugin felt natural to me, but I'm open to any better ideas!